packages/fiori-generator-shared/src/cap/index.ts

```diff
@@ -0,0 +1,75 @@
+import { existsSync, readFileSync } from 'fs';
```

Comment:
Can we use fs promises instead of sync versions?

Reply:
This function is to be sync only, right?
maybe it should be called `getCapFolderPathsSync`